osquery/tables/system/posix/augeas.cpp

```diff
@@ -15,13 +15,25 @@
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
+namespace osquery {
+
+/**
+ * @brief Augeas lenses path.
+ *
+ * Directory that contains augeus lenses.
+ */
 #ifdef __APPLE__
-#define LENSES_PATH "/private/var/osquery/lenses"
+FLAG(string,
```

Comment:
:laughing: I tried that first. Got this warning:

osquery/osquery/tables/system/posix/augeas.cpp:27:2: warning: embedding a directive within macro arguments has undefined behavior [-Wembedded-directive]
#ifdef __APPLE__
 ^
osquery/osquery/tables/system/posix/augeas.cpp:31:2: warning: embedding a directive within macro arguments has undefined behavior [-Wembedded-directive]
#endif
 ^